### ISSUE TITLE
Disable automatic updates

### ISSUE THREAD

**Issue by fubar** [kontena/lens#6420]
**What would you like to be added**:

The application is missing an option to disable automatic updates.

**Why is this needed**:

This should not need explanation, but as one of many reasons I'll offer tethering off my phone: When I'm using my cell phone data, it's extremely wasteful and expensive to download large amounts of data, and leads to unpleasant surprises when I don't even know what caused the data volumes.

This feature has been requested a couple of times before, all of such requests were closed or did not address or resolve the feature request:
- https://github.com/lensapp/lens/issues/92
- https://github.com/lensapp/lens/issues/1689

**Environment you are Lens application on:**

- Kubernetes distribution: Irrelevant
- Desktop OS: OSX

**Comment by dsimushkin** [comment]
Hey! Is there any progress on this?

Just auto-dowloaded an update while connected to my phone hotspot in roaming - that was pricy.

**Comment by HenrikDK** [comment]
Yeah +1 on getting this feature, last 2 months updates have been completely unstable we're currently staying on 6.1.19 since 6.3.0 is non working for our clusters. every morning i have to reinstall since lens seems to have updated it self without my consent.

**Comment by h0rn3t** [comment]
yup 6.3.0 is trash update broke all that working in 6.2.5

**Comment by FleetAdmiralButter** [comment]
+1 on this, 6.3.0 is unusable.

**Comment by h0rn3t** [comment]
plz disable update to 6.3.0

**Comment by huang-jy** [comment]
Agreed here also. Autoupdates are good, but should be opt-in, and not forced.

Two workarounds for others having this issue are to juggle a backup version so that the update is overwritten (e.g. something like `cp lens-6.2.5 lens`) before starting it

Or, modifying the /etc/hosts file to force github.com to resolve to 127.0.0.1 so the update check will fail (but remember to remove that edit afterwards)

**Comment by iMikeG6** [comment]
Agreed, this update 6.3.0 is crap...

Log view or exec to a pod is not visible anymore... I wondering if devs are testing their own app before releasing update as "Stable"...

On 6.2.5, no issues.
So, please, add that feature so we can decide if we want to update or not...

As for now, from Terminal, I had to chmod on the OS X OpenLens binary

```
chmod u-w /Applications/OpenLens.app/Contents/MacOS/OpenLens
```
or for Lens users
```
chmod u-w /Applications/Lens.app/Contents/MacOS/Lens
```

This should prevent Lens from updating itself as the binary is now in readonly.

As I'm also using Little snitch (Application firewall), as an extra precaution, I denied every access from the app to `github`, `k8slens.dev` or `githubusercontent.com`

With that, it's seems like there's no more update.

Devs, fix and test your app.

Thanks.

**Comment by FleetAdmiralButter** [comment]
I was also able to disable the Lens auto updater by setting the app bundle to read only using the "Get Info" panel in Finder:

![image](https://user-images.githubusercontent.com/37036084/211221890-fb483441-39a1-4fee-95ac-5724c7c58460.png)

Under "Sharing & Permissions", set yourself as "Read only" and "No Access" for everything else (might need to click the padlock).

Once Lens launches, it will ask for your admin credentials to install an update helper. Make sure to cancel out of this.

**Comment by TommasoLencioni** [comment]
@FleetAdmiralButter thanks for the workaround, it worked!

Cancelling each time the update helper installation is still tedious tho.

**Comment by huang-jy** [comment]
https://github.com/MuhammedKalkan/OpenLens/issues/81#issuecomment-1372257996 this fix worked for me:

> Hi, after installing version 6.2.5 you can block updates by changing the permissions of the pending folder:

`sudo chmod -R 000 ~/Library/Application\ Support/Caches/open-lens-updater/pending`

for Linux it's `~/.cache/open-lens-updater/pending directory`

**Comment by rblana** [comment]
on Windows I denied access on folder `%LocalAppData%\open-lens-updater\pending`

![image](https://user-images.githubusercontent.com/4211028/218973064-28280a63-e923-45ef-91b5-8922dc4923ca.png)

**Comment by littleccguy** [comment]
I am on Windows as well. I did what @rblana did, and it was working just fine. Suddenly it started updating again with 6.4.2.

**Comment by ydudarev85** [comment]
On Windows 
just delete or comment in file  \AppData\Local\Programs\OpenLens\resources\app-update.yml 
this line   
![image](https://user-images.githubusercontent.com/91878865/224933198-fdfcb954-d262-4f8e-9171-27bfaad6db68.png)
and restart OpenLens

**Comment by littleccguy** [comment]
@ydudarev85 Thanks! That did it.

**Comment by andresospina95** [comment]
the solution was easier than i though

just install latest versión 

go to file -> extensión 
![image](https://user-images.githubusercontent.com/50535711/233758475-7b48e586-cf0d-4a5e-b924-f081b68374bf.png)
and paste 
**@alebcay/openlens-node-pod-menu**
then click on install

that should solve the log and access pod menus.
